x/evm/types/msg.go

```diff
@@ -113,6 +104,14 @@ func newMsgEthereumTx(
 			Accesses: NewAccessList(tx.Accesses),
 		}
 	default:
+		txData = &LegacyTx{
```

Comment:
why is this the default now?

Reply:
dito https://github.com/evmos/evmos/pull/1815#discussion_r1346096202